generateBuildrootSBOM.py

```diff
@@ -23,7 +22,15 @@
 import csv
 import json
 
-import cyclonedx.model.bom
+import cyclonedx
```

Comment:
what a strange mix of global imports and specific imports

for example `Component` is properly imported below, but instead it is used via `cyclonedx.model.component.Component`. 
https://github.com/CycloneDX/cyclonedx-buildroot/blob/147757225f5f857373f54ba0a5df972fc58d175d/generateBuildrootSBOM.py#L60C34-L60C69

you might want to revisit the imports, usages and missed opportunities.

Reply:
I will do so over time. There is something strange going on that forces me to do the imports.